sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/api/Azure.Monitor.OpenTelemetry.Exporter.netstandard2.0.cs

```diff
@@ -20,3 +20,10 @@ public partial class AzureMonitorTraceExporter : OpenTelemetry.BaseExporter<Syst
         public override OpenTelemetry.ExportResult Export(in OpenTelemetry.Batch<System.Diagnostics.Activity> batch) { throw null; }
     }
 }
+namespace Microsoft.Extensions.Logging
+{
+    public static partial class AzureMonitorExporterLoggingExtensions
+    {
+        public static OpenTelemetry.Logs.OpenTelemetryLoggerOptions AddAzureMonitorLogExporter(this OpenTelemetry.Logs.OpenTelemetryLoggerOptions loggerOptions, System.Action<Azure.Monitor.OpenTelemetry.Exporter.AzureMonitorExporterOptions> configure = null) { throw null; }
```

Comment:
why does this extension method exist if it simply throw?

Reply:
This is an entry in API class and auto-generated using the command - `.\eng\scripts\Export-API.ps1 -ServiceDirectory monitor`